src/layout/verticalPanel/workspaceList.ts

```diff
@@ -581,12 +581,14 @@ export class WorkspaceButtonIcon extends St.Widget {
 
     msWorkspace: MsWorkspace;
     appIconList: St.Icon[];
+    desaturateEffect: Clutter.DesaturateEffect | undefined;
 
     constructor(msWorkspace: MsWorkspace) {
         super();
         this.msWorkspace = msWorkspace;
         this.appIconList = [];
-        this.add_effect(new Clutter.DesaturateEffect());
+        this.desaturateEffect = new Clutter.DesaturateEffect();
```

Comment:
This line should be deleted no?

Reply:
Absolutely correct. Good catch, sorry about that!